x-pack/plugins/fleet/server/services/artifacts/mappings.ts

```diff
@@ -82,3 +82,7 @@ export const uniqueIdFromArtifact = <
 }: T): string => {
   return `${packageName}:${identifier}-${decodedSha256}`;
 };
+
+export const uniqueIdFromId = (id: string, packageName: string): string => {
```

Comment:
nit: `uniqueIdFromId` is a bit confusing, what is the Id here?

Reply:
The id of the artifact without the packageName, so it adds the packageName to the current id as an ES id: `${packageName}:${artifactId}`.
Is the same thing `uniqueIdFromArtifact` does, but as we only have the id here I thought would be better to just create this new method instead of load the artifact to generate the unique Id with the existing method.